src/Products/PriceList.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Hyperdrive\Products;
+
+class PriceList
+{
+    const BasicFuelPrice = 1.1;

```

Comment:
Przydałoby się dodać modyfikator dostępu do tych stałych.

\+ gdybyśmy chcieli być zgodni ze standardami [PSR-1](https://www.php-fig.org/psr/psr-1/#41-constants) to wypadałoby nazywać stałe w ten sposób:
```suggestion
    const BASIC_FUEL_PRICE = 1.1;
```